signature/policy_types.go

```diff
@@ -167,6 +170,22 @@ type prSigstoreSignedFulcio struct {
 	SubjectEmail string `json:"subjectEmail,omitempty"`
 }
 
+// prSigstoreSignedPKI contains non-fulcio certificate PKI configuration options for prSigstoreSigned
+type prSigstoreSignedPKI struct {
```

Comment:
There should be _some_ public type callers can use to refer to the value. Following the other examples here, probably an interface with a private method, and public functions to construct it (`NewPRSigstoreSignedPKI` etc.) using the “functional option” pattern.